### PR TITLE
[MINOR][DOC] Fix typos in the 'configuration', 'monitoring' and 'sql-programming-guide' documentation

### PR DESCRIPTION
**Pull request by weiqingy**
## What changes were proposed in this pull request?

Fix typos in the 'configuration', 'monitoring' and 'sql-programming-guide' documentation.

## How was this patch tested?
Manually.

### CODE REVIEW

File: docs/sql-programming-guide.md [hunk 1]
```diff
@@ -1029,7 +1029,7 @@ following command:
 bin/spark-shell --driver-class-path postgresql-9.4.1207.jar --jars postgresql-9.4.1207.jar
 {% endhighlight %}
 
-Tables from the remote database can be loaded as a DataFrame or Spark SQL Temporary table using
+Tables from the remote database can be loaded as a DataFrame or Spark SQL Temporary view using
```
Comment (srowen) [inline docs/sql-programming-guide.md]: Is this definitely correct? I'm not so familiar with temp tables vs views

File: docs/monitoring.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ directory must be supplied in the `spark.history.fs.logDirectory` configuration
 and should contain sub-directories that each represents an application's event logs.
 
 The spark jobs themselves must be configured to log events, and to log them to the same shared,
-writeable directory. For example, if the server was configured with a log directory of
+writable directory. For example, if the server was configured with a log directory of
```
Comment (srowen) [inline docs/monitoring.md]: writeable is also an acceptable spelling but OK

**Review by srowen — APPROVED** [review]
OK, this is pretty minor and I'd prefer to batch together more spell-check changes. Minus one question, it's OK though

**Comment by weiqingy** [comment]
@srowen Thanks for reviewing this minor PR. I found these typo issues when I was working on my own jobs and needed to read these documentations, so the PR is pretty small. I will pay attention next time.

File: docs/sql-programming-guide.md (hunk 1 above)
Comment (weiqingy) [inline docs/sql-programming-guide.md]: Yes, thanks @gatorsmile .

**Comment by srowen** [comment]
Jenkins test this please

**Comment by srowen** [comment]
Jenkins add to whitelist

**Comment by srowen** [comment]
Merged to master/2.1

**Comment by weiqingy** [comment]
Thanks for the review, @srowen.

**Pull request closed without merging**